src/client/containers/404Page/404Page.Container.js

```diff
@@ -1,8 +1,25 @@
 import React from 'react';
 import './404Page.styles.css';
+import page404 from './../../assets/images/404.png';
+import { Link } from 'react-router-dom';
 
 function Page404() {
-  return <div>404 Page</div>;
+  return (
+    <div className="page404">
+      <br />
+      <img src={page404} alt="404" />
+      <br />
+      <br />
+      <div>Oops! The page you are looking for does not exist</div>
+      <br />
+      <Link to="/" className="link">
+        <button type="button" className="btn">
+          RETURN TO STORE
+        </button>
+      </Link>
+      <br />
```

Comment:
Hej Suvarna the page looks okay. I am not sure if it is the best practice to make so much use of <br /> tags to help your style. Maybe an div or some display block whould help.

Reply:
I agree page looks fine but still return t store bytton is sticked to footer.I think it can be resolved by footer position or wrapping the button in div.:)